repo-issues/lib/data-source.js

```diff
@@ -0,0 +1,97 @@
+const http = require('request-promise-native')
+const moment = require('moment')
+const octokit = require('@octokit/rest')()
+
+const show_detaiils_threshold = 5
+const organisation = 'novoda'
+const languageIconMap = {
```

Comment:
Icons are read from http://konpa.github.io/devicon/ but there's no exact mapping between github languages and available icons. This maps is a patch for that